core/txpool/legacypool/heap.go

```diff
@@ -0,0 +1,171 @@
+package legacypool
+
+import (
+	"container/heap"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+// txHeapItem implements the Interface interface of heap so that it can be heapified
```

Comment:
Interface interface --> interface

Reply:
The interface of heap itself is called `Interface`. So I am calling it Interface interface. Updated the comment with the link